ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/cluster-density-v2/cloud-bulldozer-e2e-benchmarking-cluster-density-v2-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
```

Comment:
Do we really need this stuff here?, it's very noisy

Reply:
Removed.